cluster/examples/kubernetes/ceph/cluster-test.yaml

```diff
@@ -17,6 +17,7 @@ data:
     [global]
     osd_pool_default_size = 1
     mon_warn_on_pool_no_redundancy = false
+    osd_crush_chooseleaf_type = 0
```

Comment:
Why do we need this in the test yaml?

Reply:
It forces failure domain = osd. I'm not sure we want to keep it or not. Maybe not since I can't think of super strong reasoning.